_sass/catalog.scss

```diff
@@ -381,17 +411,108 @@
     border-radius: 15px;
     border: 0;
     transition: 0.4s ease-out;
-    box-shadow: 2px 2px 6px 0px  #00D3A9;
+  box-shadow: 2px 2px 6px 0px #00d3a9;
   }
-   .modal-smp{
+    .modal-smp {
     width: 20px;
     bottom: 0;
     right: 0;
     vertical-align: middle;
     padding-right: 2px;
+}
+.modal-pattern-image {
+  width: 55%;
+  -o-object-fit: contain;
```

Comment:
@Rohith-Raju what does this do?

Reply:
@asubedy  "-o-object-fit: contain  = Specifies how the contents of a replaced element should be scaled relative to the box established by its used height and width.". Not entirely sure how it works. I didn't add this bust have picked up by the linter. I removed and looked for any differences, Saw none. So should I keep it or remove it

Reply:
Yes @Rohith-Raju , please look carefully for any differences if there are none remove this.